### PR TITLE
add `actor_ids` param to `create_export`

### PR DESCRIPTION
**Pull request by cmatheson**
## Description

Adds `actor_ids` / `actor_names` and deprecates `actors`

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[X] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

**Comment by robframpton** [comment]
Can we add these new filters to this unit test as well? https://github.com/workos/workos-ruby/blob/main/spec/lib/workos/audit_logs_spec.rb#L110

File: .rubocop.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ Layout/LineLength:
 Metrics/BlockLength:
   ExcludedMethods: ['describe', 'context', 'before']
 Metrics/MethodLength:
-  Max: 15
+  Max: 30
```
Comment (cmatheson) [inline .rubocop.yml:14]: we're disabling this all over the place so i've just bumped it up (lmk if i should do this in a separate PR--I realize it's making this one kind of messy).

File: lib/workos/audit_logs.rb [hunk 2]
```diff
@@ -57,10 +59,13 @@ def create_event(organization:, event:, idempotency_key: nil)
           range_end: String,
           actions: T.nilable(T::Array[String]),
           actors: T.nilable(T::Array[String]),
+          actor_names: T.nilable(T::Array[String]),
+          actor_ids: T.nilable(T::Array[String]),
           targets: T.nilable(T::Array[String]),
         ).returns(WorkOS::AuditLogExport)
       end
-      def create_export(organization:, range_start:, range_end:, actions: nil, actors: nil, targets: nil)
+      def create_export(organization:, range_start:, range_end:, actions: nil, # rubocop:disable Metrics/ParameterLists
```
[the remark this answers is not in the document]
Comment (cmatheson) [reply lib/workos/audit_logs.rb:67]: I think it should be backwards compatible since these are keyword arguments (there's no way to use them positionally afaik):
```ruby
irb(main):001:0> def foo(a:, b:); [a, b]; end
=> :foo
irb(main):002:0> foo(b: "b", a: "a")
=> ["a", "b"]
irb(main):003:0> foo(a: "a", b: "b")
=> ["a", "b"]
irb(main):004:0> foo("a", "b")
(irb):1:in `foo': wrong number of arguments (given 2, expected 0; required keywords: a, b) (ArgumentError)
```

[the remark this answers is not in the document]
Comment (cmatheson) [reply lib/workos/audit_logs.rb:67]: (still happy to swap the order though if that's preferred)

**Review by robframpton — APPROVED** [review]
